packages/devtools_app/lib/src/screens/memory/shared/primitives/painting.dart

```diff
@@ -25,13 +25,11 @@ Widget createDashWidget(Color color) {
 }
 
 Widget createSolidLine(Color color) {
-  return Padding(
+  return Container(
+    height: 6,
+    width: 20,
```

Comment:
not your fault but these should be named consts

Reply:
Agreed. FYI there is a Dart Code Metrics lint for exactly that problem we can enable next. Leaving moving these const off this CL as it is already too big and too error prone.